resources/IncludedWorkloadManifests.txt

```diff
@@ -0,0 +1,8 @@
+microsoft.net.sdk.android
```

Comment:
Rather than hard-coding this file we should be generating it automatically from the baseline manifests that we are including.

Reply:
But once we switch to the MSI manifest installers, that would become more difficult, unless we continue to take both the manifest nupkg and the manifest msi nupkg

Reply:
We'll need the manifest .nupkg to create the zip layout of the SDK, right?

Reply:
Yes, that's a good point :)

Reply:
Cool, I'll update to generating this file then